library/core/src/panic/panic_info.rs

```diff
@@ -161,11 +161,12 @@ impl fmt::Display for PanicInfo<'_> {
     fn fmt(&self, formatter: &mut fmt::Formatter<'_>) -> fmt::Result {
         formatter.write_str("panicked at ")?;
         self.location.fmt(formatter)?;
+        formatter.write_str(":")?;
```

Comment:
note that this inflates the number of write calls by 1 on normal panic paths, compared to an else branch

Reply:
Yeah, and it avoids code duplication.